tekton/release-pipeline.yaml

```diff
@@ -54,6 +54,16 @@ spec:
     - name: git-clone
       taskRef:
         name: git-clone
+        resolver: git
+        params:
+        - name: repo
+          value: pipeline
+        - name: org
+          value: tektoncd
+        - name: revision
+          value: $(params.gitRevision)
+        - name: pathInRepo
+          value: tekton/publish.yaml
```

Comment:
The git-clone task is defined in the catalog, not in the pipeline repo.
We could either pull it from there with the git resolver and pin it to a revision that matches the current sha of the tip of the `main` branch. Alternatively we could use the bundle remote resolver and consume it, as we do for in the plumbing repo, from the bundles published to grc.io - see https://github.com/tektoncd/plumbing/blob/2b2d5d06c8ffd885945feae76318ac8a153b3c8d/tekton/ci/jobs/tekton-golang-tests.yaml#L37-L44 for an example

Reply:
Thanks so much Andrea! This was very helpful. 
I've used the first approach to update the release pipeline. Any feedback would be greatly appreciated! 😊🙏

Reply:
Should we also include `package` in params? Like in the example for bundles, https://github.com/tektoncd/catalog?

```
params:
  - name: package
    value: github.com/tektoncd/pipeline
```

Reply:
Should I also include `package` in params? Like in the example for bundles, https://github.com/tektoncd/catalog?

```
params:
  - name: package
    value: github.com/tektoncd/pipeline
```